freshmaker/handlers/koji/rebuild_flatpak_application_on_module_ready.py

```diff
@@ -217,6 +215,10 @@ def _filter_images_with_higher_rpm_nvr(self, images):
             private_key=conf.lightblue_private_key,
             event_id=self.current_db_event_id,
         )
+
+        if not errata_rpm_nvrs:
+            errata_rpm_nvrs = None
+
         images = lb.get_images_by_nvrs(images, rpm_nvrs=errata_rpm_nvrs)
```

Comment:
Can you fix this in `get_images_by_nvrs()` instead? It seems it should check for empty list instead of None.

Reply:
Actually, this should be probably just:
```python
        if errata_rpm_nvrs:
            images = lb.get_images_by_nvrs(images, rpm_nvrs=errata_rpm_nvrs)
```

Reply:
Ack. Fixed

Reply:
This is wrong actually, maybe I should use different variable names.
return value images is ContainerImage, but input images is a list of NVRS(str)
Let me correct it, @hluk Please have a look again.